package-lock.json

```diff
@@ -0,0 +1,15344 @@
+{
```

Comment:
vai dar merda quando for subir em produção se esse arquivo existir, já passei por isso, melhor apagar